tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -611,3 +611,38 @@ stages:
                 ls -laR ${{ variables.memoryUsageTestOutputFolder }};
                 echo "Cleanup  ${{ variables.memoryUsageTestOutputFolder }}"
                 rm -rf ${{ variables.memoryUsageTestOutputFolder }};
+
+  - stage: runAfterAll
+    condition: always()
+    dependsOn:
+      - perf_unit_tests_runtime
+      - perf_unit_tests_memory
+      - perf_e2e_tests_local
+      - perf_e2e_tests_odsp
+
+    jobs:
+      - template: templates/include-test-perf-benchmarks.yml
+        parameters:
+          poolBuild: ${{ parameters.poolBuild }}
+          testWorkspace: ${{ variables.testWorkspace }}
+          customSteps:
+
+          - task: Bash@3
+            displayName: Retrieve buildId results
+            inputs:
+              targetType: 'inline'
+              workingDirectory: $(toolAbsolutePath)
+              script: |
+                echo "creating output folder"
+                mkdir -p ${{ variables.testWorkspace }}/timingOutput
+                echo "Executing curl command ..."
+                echo  'curl -u ":$(System.AccessToken)" "https://dev.azure.com/fluidframework/internal/_apis/build/builds/$(Build.BuildId)/timeline"'
```

Comment:
don't log credentials

Reply:
they show up as ***  , so I assume we should be fine.
Ex: 
curl -u ":***" "https://dev.azure.com/***framework/internal/_apis/build/builds/162093/timeline"
https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_apis/build/builds/162093/logs/150